src/main/java/uk/gov/hmcts/reform/et/syaapi/service/PdfMapperService.java

```diff
@@ -0,0 +1,325 @@
+        printFields.putAll(printRepresentative(caseData.getRepresentativeClaimantType()));
+        return printFields;
+    }
+
+    private void verifyCase(CaseData caseData) throws PdfMapperException {
+        if (caseData == null) {
+            throw new PdfMapperException("pdfMapperService: Case provided was null");
+        }
+    }
+
+    private Map<String, String> printPersonalDetails(CaseData caseData) {
+        Map<String, String> printFields = createHashMap();
+        if (!TITLES.get(caseData.getClaimantIndType().getClaimantTitle()).isEmpty()) {
+            printFields.put(
+                TITLES.get(caseData.getClaimantIndType().getClaimantTitle()),
+                TITLE_MAP.get(caseData.getClaimantIndType().getClaimantTitle())
+            );
+        }
+        printFields.put(PdfMapperConstants.Q1_FIRST_NAME, caseData.getClaimantIndType().getClaimantFirstNames());
+        printFields.put(PdfMapperConstants.Q1_SURNAME, caseData.getClaimantIndType().getClaimantLastName());
+        LocalDate dob = LocalDate.parse(caseData.getClaimantIndType().getClaimantDateOfBirth());
+        printFields.put(PdfMapperConstants.Q1_DOB_DAY, StringUtils.leftPad(String.valueOf(dob.getDayOfMonth()),
+            2, "0"));
+        printFields.put(PdfMapperConstants.Q1_DOB_MONTH, StringUtils.leftPad(String.valueOf(dob.getMonthValue()),
+            2, "0"));
+        printFields.put(PdfMapperConstants.Q1_DOB_YEAR, String.valueOf(dob.getYear()));
+        printFields.put(PdfMapperConstants.Q1_SEX, caseData.getClaimantIndType().getClaimantSex());
+        printFields.put(String.format(PdfMapperConstants.QX_HOUSE_NUMBER, CLAIMANT_ADDRESS_PREFIX),
+            caseData.getClaimantType().getClaimantAddressUK().getAddressLine1());
+        printFields.put(String.format(PdfMapperConstants.QX_STREET, CLAIMANT_ADDRESS_PREFIX),
+            caseData.getClaimantType().getClaimantAddressUK().getAddressLine2());
+        printFields.put(String.format(PdfMapperConstants.QX_POST_TOWN, CLAIMANT_ADDRESS_PREFIX),
+            caseData.getClaimantType().getClaimantAddressUK().getPostTown());
+        printFields.put(String.format(PdfMapperConstants.QX_COUNTY, CLAIMANT_ADDRESS_PREFIX),
+            caseData.getClaimantType().getClaimantAddressUK().getCounty());
+        printFields.put(String.format(PdfMapperConstants.QX_POSTCODE, CLAIMANT_ADDRESS_PREFIX),
+            caseData.getClaimantType().getClaimantAddressUK().getPostCode());
+        printFields.put(String.format(PdfMapperConstants.QX_PHONE_NUMBER, "1.6"),
+            caseData.getClaimantType().getClaimantPhoneNumber());
+        printFields.put(PdfMapperConstants.Q1_MOBILE_NUMBER, caseData.getClaimantType().getClaimantMobileNumber());
+        printFields.put(PdfMapperConstants.Q1_EMAIL, caseData.getClaimantType().getClaimantEmailAddress());
+        String contactPreference = caseData.getClaimantType().getClaimantContactPreference();
+        if ("Email".equals(contactPreference)) {
```

Comment:
You've added constants for Yes and No. Shouldn't you do the same for Email and Post?